Templates/admin menu.php

```diff
@@ -39,5 +39,5 @@
 			?>
 		</section>
 	</div>
-	Wuwana v2.1.5
+	<div id="version"> Wuwana v2.1.6 - text-css </div>
```

Comment:
Why did you add "text-css" ??

Reply:
to know which was the latest branch. Is it not a good idea?

Reply:
No